client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx

```diff
@@ -207,7 +207,7 @@ const InvestigationTable: React.FC = (): JSX.Element => {
         switch (cellName) {
             case TableHeadersNames.color:
                 return (
-                    indexedRow.groupId ?
+                    indexedRow.canFetchGroup ?
```

Comment:
why did you changed this condition ?
what its trying to check ?

Reply:
I've noticed in another locaiton in the code that ```canFetchGroup``` exists 
this is it's declatation
![image](https://user-images.githubusercontent.com/68457306/102018829-62546500-3d78-11eb-82d2-9c68a83bbdd4.png)
It's just a boolean representation of if groupId (which is a number) exists.
I think this is a more _typescript_ appropriate approach to this condition

Reply:
cool

Reply:
OK , So turns that when the row is a fetched row (linked row from the table) the value of canFetchGroup is false - so i changed it to Boolean(indexedRow.groupId) in the next commit 😞